File: Deafios em C#/1115 - Quadrante.cs
using System;

//Developed by: @LucasMarcuzo
class URI1115 {
    static void Main() {
   
        float x, y, z=0.0f;

        do{
            string[] array = Console.ReadLine().Split(' ');

            x=float.Parse(array[0]);
            y=float.Parse(array[1]);

            if(x > z && y > z){

                Console.WriteLine("primeiro");

            }else if(x < z && y > z){

                Console.WriteLine("segundo");

            }else if(x < z && y < z){

                Console.WriteLine("terceiro");
        
            }else if(x > z && y < z){

                Console.WriteLine("quarto");
            }
        
        }while(x != 0 && y != 0);
    
    }
}

File: Desafios em JavaScript/1115 - Quadrante.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var x, y, z=0.0;

do{

    let ln = lines.shift().split(` `);
    x=parseInt(ln.shift());
    y=parseInt(ln.shift());

    if(x > z && y > z){

        console.log("primeiro");

    }else if(x < z && y > z){

        console.log("segundo");

    }else if(x < z && y < z){

        console.log("terceiro");

    }else if(x > z && y < z){

        console.log("quarto");
    }

}while(x !== 0 && y !== 0);
